openff/interchange/components/interchange.py

```diff
@@ -367,6 +385,59 @@ def to_openmm_topology(
             ensure_unique_atom_names=ensure_unique_atom_names,
         )
 
+    def to_openmm_simulation(
+        self,
+        integrator: "openmm.Integrator",
+        combine_nonbonded_forces: bool = True,
+        add_constrained_forces: bool = False,
+        **kwargs,
+    ) -> "openmm.app.Simulation":
```

Comment:
```suggestion
    ) -> "openmm.app.simulation.Simulation":
```

This is the magic incantation needed to let Sphinx find the documentation link :/